ui/src/app/pages/access/access-renewal.tsx

```diff
@@ -218,44 +227,244 @@ const BackArrow = withCircleBackground(() => (
   <Arrow style={{ height: 21, width: 18 }} />
 ));
 
+export const RenewalCardBody = (props: {
+  moduleStatus: AccessModuleStatus;
+  setLoading: (boolean) => void;
+  hide?: boolean;
+  textStyle?: CSSProperties;
+  showTimeEstimate?: boolean;
+}) => {
+  const {
+    moduleStatus,
+    setLoading,
+    hide,
+    textStyle,
+    showTimeEstimate = false,
+  } = props;
+
+  const [, navigateByUrl] = useNavigation();
+  const noReportId = useId();
+  const reportId = useId();
+  const [publications, setPublications] = useState<boolean>(null);
+  const [trainingRefreshButtonDisabled, setTrainingRefreshButtonDisabled] =
+    useState(true);
+
+  const { AARTitleComponent, renewalTimeEstimate } = getAccessModuleConfig(
+    moduleStatus.moduleName
+  );
+  const { lastConfirmedDate, nextReviewDate } =
+    computeRenewalDisplayDates(moduleStatus);
+  const TimeEstimate = () =>
+    showTimeEstimate ? (
+      <div>
+        <span style={{ padding: 10 }}>
+          <Clock style={{ color: colors.disabled }} />
+        </span>
+        {renewalTimeEstimate} min
+      </div>
+    ) : null;
+
+  const Dates = () => (
+    <div style={{ ...renewalStyle.dates, ...textStyle }}>
+      <div>Last Updated On:</div>
+      <div>Next Review:</div>
+      <div>{lastConfirmedDate}</div>
+      <div>{nextReviewDate}</div>
+    </div>
+  );
+
+  const module = switchCase(
+    moduleStatus.moduleName,
+    [
+      AccessModule.PROFILECONFIRMATION,
+      () => (
+        <React.Fragment>
+          <Dates />
+          <div style={{ marginBottom: '0.5rem', ...textStyle }}>
+            Please update your profile information if any of it has changed
+            recently.
+          </div>
+          <div style={textStyle}>
+            Note that you are obliged by the Terms of Use of the Workbench to
+            keep your profile information up-to-date at all times.
+          </div>
+          <TimeEstimate />
+          <ActionButton
+            actionButtonText='Review'
+            completedButtonText='Confirmed'
+            moduleStatus={moduleStatus}
+            onClick={() =>
+              navigateByUrl('profile', { queryParams: { renewal: 1 } })
+            }
+          />
+        </React.Fragment>
+      ),
+    ],
+    [
+      AccessModule.PUBLICATIONCONFIRMATION,
+      () => (
+        <React.Fragment>
+          <Dates />
+          <div style={textStyle}>
+            The <AoU /> Publication and Presentation Policy requires that you
+            report any upcoming publication or presentation resulting from the
+            use of <AoU /> Research Program Data at least two weeks before the
+            date of publication. If you are lead on or part of a publication or
+            presentation that hasn’t been reported to the program,{' '}
+            <a
+              target='_blank'
+              style={{ textDecoration: 'underline' }}
+              href={'https://redcap.pmi-ops.org/surveys/?s=MKYL8MRD4N'}
```

Comment:
nit: i wonder if href should be constant

Reply:
done